stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/mapping/ChannelMapping.kt

```diff
@@ -21,16 +21,20 @@ import io.getstream.chat.android.client.extensions.syncUnreadCountWithReads
 import io.getstream.chat.android.models.Channel
 import io.getstream.chat.android.models.User
 import io.getstream.chat.android.models.UserId
+import java.util.Date
 
-internal fun DownstreamChannelDto.toDomain(currentUserId: UserId?): Channel =
+internal fun DownstreamChannelDto.toDomain(
+    currentUserId: UserId?,
+    eventChatLastMessageAt: Date?,
```

Comment:
Quick question for my better understanding: Is it expected that the `DownstreamChannelDto.last_message_at` field can be outdated, and we should prioritise the `eventChatLastMessageAt`? Looks a bit strange that we can get 2 different timestamps with in the same event (one in the channel model, and one in the event)

Reply:
Ideally, it should be the same, but just to ensure we are using the more updated one (The event was created right now, but the channel could be cached somehow) and to avoid any verification in our business logic, I decided to add it here, on the part where the API is converted to our business domain.
It is only overridden when the channel is parsed within a `ChatEvent`

Reply:
Thank you for the explanation!